facade/dfs.go

```diff
@@ -637,3 +637,9 @@ func (info *registryVersionInfo) start(isvcsRoot string, hostPort string) (*dock
 
 	return container, nil
 }
+
+// DockerOverride will replace a docker image in the registry with a new image for all services that use
+//  it under the given tenant
```

Comment:
Nope, leftover from an earlier approach.  I'll remove the part about a given tenant.